### PR TITLE
docker-machine 0.6.0

### PR DESCRIPTION
**Pull request by CWSpear**
This one was easier =)

### CODE REVIEW

**Comment by deekthesqueak** [comment]
Line 9 (revision 1) should be removed as there is is a minor version change. Revision only gets added when the version doesn't change. See Formulae Revisions in https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md

File: Library/Formula/docker-machine.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ class DockerMachine < Formula
   desc "Create Docker hosts locally and on cloud providers"
   homepage "https://docs.docker.com/machine"
   url "https://github.com/docker/machine.git",
-    :tag => "v0.5.6",
-    :revision => "61388e98540321b34f4a27f88df9e7a4443b9ac8"
+    :tag => "v0.6.0",
+    :revision => "e27fb87286cb8e7454183ce46a6e1e84b31965e9"
   revision 1
```
Comment (deekthesqueak) [inline Library/Formula/docker-machine.rb]: Line 9 (revision 1) should be removed as there is is a minor version change. Revision only gets added when the version doesn't change. See Formulae Revisions in https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md

Comment (CWSpear) [inline Library/Formula/docker-machine.rb]: Updated!

**Comment by DomT4** [comment]
Merged https://github.com/Homebrew/homebrew/pull/48858#issuecomment-180467595 instead for the reasoning in that comment. Thank you for the submission though! :heart:

**Comment by CWSpear** [comment]
No worries!

**Pull request closed without merging**